tests/src/test/java/alluxio/master/journal/ufs/UfsJournalIntegrationTest.java

```diff
@@ -247,6 +247,9 @@ private void deleteTestUtil() throws Exception {
     registry.stop();
   }
 
+  /**
+   * Tests journalling of empty file system.
```

Comment:
instead of adding this javadoc, could we rename the test method to `emptyFileSystem`?